donate/settings.py

```diff
@@ -19,14 +19,15 @@ class Config:
     LOG_FORMAT = "%(asctime)s | %(pathname)s:%(lineno)d |" \
         "%(funcName)s | %(levelname)s | %(message)s "
     SINGLE_CCY = True
+    CSRF_ENABLED = True
 
 
 class ProdConfig(Config):
     """ Production Configuration """
 
     ENV = 'prod'
     DEBUG = False
-    SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL',
+    SQLALCHEMY_DATABASE_URI = os.environ.get('PROD_DATABASE_URI',
                                              'postresql://localhost/example')
```

Comment:
It might be good to print a warning / error here if the database URI isn't set instead of silently using an example URI.

Reply:
If they are None an error is now raised when creating the app in autoapp.py